sdk/identity/azure-identity/vcpkg/Config.cmake.in

```diff
@@ -4,7 +4,7 @@
 @PACKAGE_INIT@
 
 include(CMakeFindDependencyMacro)
-find_dependency(azure-core-cpp)
+find_dependency(azure-core-cpp "1.2.0")
```

Comment:
Or maybe we can release 1.1.1 with just the bug fixes without it going to the beta phase? New features will go thru 1.2.0 beta.

Reply:
I think we can release 1.2.0 GA directly.  Actually we need to, because I am planning to release KeyVault Keys GA depending on azure core 1.2.0 GA

Reply:
👍 Let's also update the corresponding `find_package`/`find_dependency`/`version>=` for the KeyVault then.